cli/remotebazel/remotebazel.go

```diff
@@ -489,6 +489,12 @@ func streamLogs(ctx context.Context, bbClient bbspb.BuildBuddyServiceClient, inv
 	var chunks []*elpb.GetEventLogChunkResponse
 	wasLive := false
 	for {
+		select {
+		case <-stopChan:
+			return nil
+		default:
+		}
+
```

Comment:
instead of having a separate channel wdyt about using context cancellation? `signal.NotifyContext()` might be useful but if it winds up complicating things then nbd.